src/services/storage/models/offer.model.ts

```diff
@@ -112,3 +107,23 @@ export function getBillingPriceAvgQuery (
   )
   `)
 }
+
+/**
+ * This function generates nested query for available capacity for offer
+ */
+export function getAvailableCapacityQuery (): Literal {
+  return literal(`
+    totalCapacity - COALESCE(
+      (SELECT
+        CAST(
+          SUM(
+            "storage_agreement"."size"
+            ) as STRING
```

Comment:
Hmm... You should wrap the whole thing with `CAST as String` instead of wrapping just availableCapacity, as we had totalCapacity(Number) - availableCapacity(String)